packages/components/link/src/link.tsx

```diff
@@ -127,7 +127,7 @@ const getLinkStyles = (props: TLinkProps & { isNewTheme: boolean }) => {
 const Wrapper = styled.span`
   > svg {
     margin: 0 0 0 ${designTokens.spacing10} !important;
-    vertical-align: bottom;
+    vertical-align: ${designTokens.verticalAlignForLinkIcon};
```

Comment:
I was wondering whether we actually need a token. When I check the updated story in the preview deployment I see using `middle` in the default theme also seems to work, but maybe I'm missing something.

Reply:
Thanks, that's a solid point 👍
I actually wanted to bring it up tomorrow. It seems fine from my side as well but I also wanted to double check with everyone.